nrf_modem/doc/sockets/socket_options_func.rst

```diff
@@ -17,23 +17,25 @@ Following is a list of supported functions:
 
 * :c:func:`nrf_socket`
 * :c:func:`nrf_close`
+* :c:func:`nrf_fcntl`
```

Comment:
How did you re-arrange this list?

Reply:
Moved re-arranging to separate commit.

Reply:
I think the order of those functions in `nrf_socket.h` could be improved, it seems a bit random.

Reply:
Yes, agree.